### PR TITLE
Report limited editors to admin and only to admin

### PR DESCRIPTION
**Pull request by yvanzo**

### CODE REVIEW

File: root/report/index.tt [hunk 1]
```diff
@@ -29,6 +29,14 @@
             <li><a href="[% c.uri_for_action('/report/show', 'ArtistsDisambiguationSameName') %]">[% l('Artists with disambiguation the same as the name') %]</a></li>
         </ul>
 
+        [% IF c.user.is_account_admin -%]
```
Comment (mwiencek) [inline root/report/index.tt]: Other than not outputting the link here, where does it restrict access to account admins?

Comment (yvanzo) [reply root/report/index.tt]: Nowhere else. Would you prefer it to be done in https://github.com/metabrainz/musicbrainz-server/pull/556/files#diff-068aefca9b2fdb274a6a5cd3cb07a5e2 ?

Comment (yvanzo) [reply root/report/index.tt]: Oops, fixed in https://github.com/metabrainz/musicbrainz-server/pull/556/commits/cc1a4fdc419081b809f511d471435208a00b0d5e#diff-c720a88e431a4eb472afdd86f5d7206fR12

**Comment by mwiencek** [comment]
#557 changes the user-visible name to "new" editor, but I think it's fine to keep "limited" here.

File: lib/MusicBrainz/Server/Report/LimitedEditors.pm [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package MusicBrainz::Server::Report::LimitedEditors;
+use Moose;
+
+use MusicBrainz::Server::Constants qw( $EDITOR_MODBOT );
+
+with 'MusicBrainz::Server::Report::EventReport';
```
Comment (mwiencek) [inline lib/MusicBrainz/Server/Report/LimitedEditors.pm]: This looks unnecessary.

File: lib/MusicBrainz/Server/Report/LimitedEditors.pm [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+SELECT id, name, email, website, bio, member_since, email_confirm_date,
+       last_login_date, last_updated, deleted,
+       row_number() OVER (ORDER BY id DESC)
+  FROM editor eor
+ WHERE id != $EDITOR_MODBOT
+   AND NOT deleted
+   AND   ( email_confirm_date IS NULL
+        OR member_since < NOW() - INTERVAL '2 weeks'
+        OR (SELECT COUNT(*) FROM edit WHERE eor.id = edit.editor AND edit.status = 2 AND edit.autoedit = 0) < 10
+         )";
```
Comment (mwiencek) [inline lib/MusicBrainz/Server/Report/LimitedEditors.pm]: Is it currently unordered, or ordered by name elsewhere? Should we order by `member_since DESC` instead? It seems like it'd be useful for admins to see the most recent accounts at the top of the list.

Comment (mwiencek) [reply lib/MusicBrainz/Server/Report/LimitedEditors.pm]: I guess @Freso or @reosarevok will be looking at this report the most, so however they'd want it ordered.

Comment (yvanzo) [reply lib/MusicBrainz/Server/Report/LimitedEditors.pm]: Yes. Actually, rows are ordered by `id` at [line 11](https://github.com/metabrainz/musicbrainz-server/pull/556/files/cc1a4fdc419081b809f511d471435208a00b0d5e#diff-d1da13642c99eb0de1f985a77a2c6467R11). Greatest ID being the most recent, it is the same as ordering by `member_since`.

File: lib/MusicBrainz/Server/Report/LimitedEditors.pm (hunk 2 above)
Comment (yvanzo) [reply lib/MusicBrainz/Server/Report/LimitedEditors.pm]: Fixed in https://github.com/metabrainz/musicbrainz-server/pull/556/commits/125982facf0c08a9f6f34d581c15fbd36f513416

**Pull request merged**